vulture_os/gui/urls.py

```diff
@@ -37,7 +37,7 @@
 
     path('collapse', collapse, name="gui.collapse_menu"),
 
-    re_path(r'^$', dashboard.dashboard_services, name="gui.dashboard.services"),
+    path('', dashboard.dashboard_services, name="gui.dashboard.services"),
```

Comment:
not sure I want to make this change

Reply:
That's not mandatory, it was performed by django-upgrade and seemed relevant for me.
I can revert this change if needed.

Source : https://github.com/adamchainz/django-upgrade#urls